src/main/java/org/apache/sling/discovery/impl/DiscoveryServiceConfig.java

```diff
@@ -0,0 +1,138 @@
+    String[] DEFAULT_TOPOLOGY_CONNECTOR_WHITELIST = {"localhost", "127.0.0.1"};
+
+    /** Path of resource where to keep discovery information, e.g /var/discovery/impl/ */
+    @AttributeDefinition(name = "%discoveryResourcePath.name", description = "%discoveryResourcePath.description")
+    String discoveryResourcePath() default DEFAULT_DISCOVERY_RESOURCE_PATH;
+    String DEFAULT_DISCOVERY_RESOURCE_PATH = "/var/discovery/impl/";
+
+    /**
+     * If set to true, local-loops of topology connectors are automatically stopped when detected so.
+     */
+    @AttributeDefinition(name = "%autoStopLocalLoopEnabled.name", description = "%autoStopLocalLoopEnabled.description")
+    boolean autoStopLocalLoopEnabled() default false;
+
+    /**
+     * If set to true, request body will be gzipped - only works if counter-part accepts gzip-requests!
+     */
+    @AttributeDefinition(name = "%gzipConnectorRequestsEnabled.name", description = "%gzipConnectorRequestsEnabled.description")
+    boolean gzipConnectorRequestsEnabled() default false;
+
+    /**
+     * If set to true, hmac is enabled and the white list is disabled.
+     */
+    @AttributeDefinition(name = "%hmacEnabled.name", description = "%hmacEnabled.description")
+    boolean hmacEnabled() default true;
+
+    /**
+     * If set to true, and the whitelist is disabled, messages will be encrypted.
+     */
+    @AttributeDefinition()
+    boolean enableEncryption() default false;
+
+    /**
+     * The value for the shared key, shared amongst all instances in the same cluster.
+     */
+    @AttributeDefinition(name = "%sharedKey.name", description = "%sharedKey.description")
+    String sharedKey();
+
+    /**
+     * The default lifetime of a HMAC shared key in ms. (4h)
+     */
+    long DEFAULT_SHARED_KEY_INTERVAL = 3600L*1000*4;
+    @AttributeDefinition(name = "%hmacSharedKeyTTL.name", description = "%hmacSharedKeyTTL.description")
+    long hmacSharedKeyTTL() default DEFAULT_SHARED_KEY_INTERVAL;
+
+    /**
+     * The property for defining the backoff factor for standby (loop) connectors
+     */
+    @AttributeDefinition(name = "%backoffStandbyFactor.name", description = "%backoffStandbyFactor.description")
+    int backoffStandbyFactor() default DEFAULT_BACKOFF_STANDBY_FACTOR;
+    int DEFAULT_BACKOFF_STANDBY_FACTOR = 5;
+
+    /**
+     * The property for defining the maximum backoff factor for stable connectors
+     */
+    @AttributeDefinition(name = "%backoffStableFactor.name", description = "%backoffStableFactor.description")
+    int backoffStableFactor() default DEFAULT_BACKOFF_STABLE_FACTOR;
+    int DEFAULT_BACKOFF_STABLE_FACTOR = 5;
+
+    /**
+     * when set to true and the syncTokenService (of discovery.commons) is available,
+     * then it is used
+     */
+    @AttributeDefinition(name = "%useSyncTokenServiceEnabled.name", description = "%useSyncTokenServiceEnabled.description")
+    boolean useSyncTokenServiceEnabled() default true;
```

Comment:
The name of this property changed from `useSyncTokenService` to `useSyncTokenServiceEnabled`.

Reply:
Good catch. I will create a PR and correct this.

Reply:
See https://github.com/apache/sling-org-apache-sling-discovery-impl/pull/8